scripts/cluster_settings

```diff
@@ -1,9 +1,9 @@
 . "${SCRIPTS_DIR}"/lib/source_only
 
 # For shipyard we need a very minimal setup to verify the deployment works
-cluster_nodes['cluster1']="control-plane"
-cluster_nodes['cluster2']="control-plane worker"
-cluster_nodes['cluster3']="control-plane worker"
+clusters=(cluster1)
```

Comment:
I don't think this is enough, we should at least validate `deploy` target works and deploys subm properly (with broker and all the other bells & whistles).

If you really want a super minimal deployment for e2e you can have a e2e_cluster_settings or some such thing.

Reply:
yes - I was thinking the same thing when I realized the CI tests various deploy options. I plan to have 2 `cluster_settings`: one with 3 clusters for CI as it was before and the other with 1 cluster for e2e.

Reply:
Done